frontend/src/__tests__/cypress/cypress/tests/e2e/dataSciencePipelines/createRunDeletePipelineCustomPipMirror.cy.ts

```diff
@@ -0,0 +1,123 @@
+import { deleteOpenShiftProject } from '~/__tests__/cypress/cypress/utils/oc_commands/project';
+import { HTPASSWD_CLUSTER_ADMIN_USER } from '~/__tests__/cypress/cypress/utils/e2eUsers';
+import { projectListPage, projectDetails } from '~/__tests__/cypress/cypress/pages/projects';
+import { pipelineImportModal } from '~/__tests__/cypress/cypress/pages/pipelines/pipelineImportModal';
+import { createRunPage } from '~/__tests__/cypress/cypress/pages/pipelines/createRunPage';
+import { pipelineRunsGlobal } from '~/__tests__/cypress/cypress/pages/pipelines/pipelineRunsGlobal';
+import {
+  pipelinesGlobal,
+  pipelineDeleteModal,
+} from '~/__tests__/cypress/cypress/pages/pipelines/pipelinesGlobal';
+import { pipelinesTable } from '~/__tests__/cypress/cypress/pages/pipelines/pipelinesTable';
+import {
+  pipelineDetails,
+  pipelineRunDetails,
+} from '~/__tests__/cypress/cypress/pages/pipelines/topology';
+import { provisionProjectForPipelines } from '~/__tests__/cypress/cypress/utils/pipelines';
+import { getIrisPipelinePath } from '~/__tests__/cypress/cypress/utils/fileImportUtils';
+import { createOpenShiftConfigMap } from '~/__tests__/cypress/cypress/utils/oc_commands/configmap';
+
+const projectName = 'test-dsp-custom-pip-prj';
+const dspaSecretName = 'test-custom-pip-dspa-secret';
+const testPipelineIrisName = 'test-iris-pipeline';
+const testRunName = 'test-pipelines-run';
+
+describe('An admin user can import and run a pipeline', { testIsolation: false }, () => {
+  before(() => {
+    provisionProjectForPipelines(projectName, dspaSecretName);
+    const pipConfig = Cypress.env('PIP_CONFIG');
+    //Create Pipelines ConfigMap With Custom Pip Index Url And Trusted Host
+    createOpenShiftConfigMap('ds-pipeline-custom-env-vars', projectName, {
+      // The following lines should be snake case
+      /* eslint-disable-next-line camelcase */
```

Comment:
Should these eslint lines be in here?

Reply:
Yes, those are in order to exclude one rule of the linter, as the variable name should be snake case